itest_src/datasets/case01/data/schema/Filter.schema

```diff
@@ -1,6 +1,6 @@
 -- MySQL dump 10.13  Distrib 5.1.61, for redhat-linux-gnu (x86_64)
 --
--- Host: localhost    Database: qservTest_case01_m
+-- Host: localhost    Database: case01_m
 -- ------------------------------------------------------
 -- Server version	5.1.61-log
```

Comment:
Just a minor comment. These original comments in the schema files no longer make any sense. It doesn't matter which database, MySQL version, etc. was used as a source for the schema as we will never be able to recover the original context. I propose cleaning up these comments in this and similar files.

This isn't really urgent. It could be doen any time later in a separate JIRA ticket.